### ISSUE TITLE
Focuses out when removing error text

### ISSUE THREAD

**Issue by musaffa** [dockyard/ember-easy-form#219]
In a form page like this:
```haml
= form-for resource
  = input-for 'title'
  = input-for 'body'
  = submit as='button'
```

I've a presence validation for title. When I focus out of the title field, an error message is displayed. When I start writing in the title input field, after taking only one character, it removes the message but at the same time focuses out of the title field. For the fields containing multiple validations, it focuses out of the input fields every time an error text removal is triggered.

I'm using ember-cli 1.13.13, ember 2.0 and ember-easy-form `1-0-stable` branch (the most recent as of this writing).

**Comment by diogomafra** [comment]
Ember is re-rendering the component when the error changes, and then the component loses focus.

Do you want to try to fix it? I will not be able to work on this for the next few weeks.

**Comment by musaffa** [comment]
I'll try to. Can you give me some hints where to look for to fix it?

**Comment by diogomafra** [comment]
First I would search for similar issues reported in the Ember's repo. I'm sure someone else already had this problem. Maybe this gives some indication of what is the ocurring.

**Comment by diogomafra** [comment]
I think I found a solution. :smile: Could you check if it's working in your project?

Replace the code of the file `addon/keywords/input-for.js` with this:

```js
import {assign} from 'ember-easy-form/utilities';
import {knownProperties} from 'ember-easy-form/components/internal-input-for';

export default function inputFor(morph, env, scope, originalParams, hash, template, inverse, visitor) {

  // Move all values to the `inputOptions` hash, except the ones we use in the `internal-input-for`
  let customHash = {};
  let inputOptions = assign({}, hash);
  for(let i=0; i<knownProperties.length; i++) {
    const key = knownProperties[i];
    if (inputOptions.hasOwnProperty(key)) {
      customHash[key] = inputOptions[key];
      delete inputOptions[key];
    }
  }

  customHash.inputOptions = inputOptions;

  env.hooks.component(morph, env, scope, 'internal-input-for', originalParams, customHash, { default: template, inverse }, visitor);
  return true;
}
```

**Comment by musaffa** [comment]
Yes it works. :+1:

**Comment by diogomafra** [comment]
Fixed: #220
